CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 - Fixed `RouteOptions#toUrl` for a case when `RouteOptions` was deserialized from a json generated by an old version of mapbox-java. [#1447](https://github.com/mapbox/mapbox-java/pull/1447)
-
+- Added `tollCost` to `RouteOptions` since it is a route request parameter. [#1458](https://github.com/mapbox/mapbox-java/pull/1458)
```

Comment:
a changelog item is not needed as we keep these changes undocumented cc @zugaldia